### PR TITLE
First stepts for update logic

### PR DESCRIPTION
**Pull request by SirDegraf**
Implements:
- https://github.com/code-kern-ai/refinery/issues/95

PRs:
- https://github.com/code-kern-ai/refinery-submodule-model/pull/27
- https://github.com/code-kern-ai/refinery-gateway/pull/86
- https://github.com/code-kern-ai/refinery-ui/pull/95
- https://github.com/code-kern-ai/refinery-updater/pull/17

### CODE REVIEW

File: upgrade_logic/business_objects/gateway.py [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+def __gateway_1_6_1_add_attribute_visibility() -> bool:
+    sql = f"""
+    UPDATE attribute
+    SET visibility = '{enums.AttributeVisibility.DO_NOT_HIDE.value}'
+    WHERE visibility IS NULL
+    """
+    general.execute_sql(sql)
+    general.commit()
+    return True
```
Comment (JWittmeyer) [inline upgrade_logic/business_objects/gateway.py]: we usually include an information in the log if something was changed 
if you don't want to do this with the orm model counting first for IS NULL should do the trick

Comment (SirDegraf) [reply upgrade_logic/business_objects/gateway.py]: Added count, check and print statement

File: upgrade_logic/business_objects/gateway.py [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+
+def gateway_1_6_1() -> bool:
+    __gateway_1_6_1_add_attribute_visibility()
+    return True
+
+
+def __gateway_1_6_1_add_attribute_visibility() -> bool:
+    print(
+        f"Set visibility of attributes to  value {enums.AttributeVisibility.DO_NOT_HIDE.value}",
```
Comment (JWittmeyer) [inline upgrade_logic/business_objects/gateway.py]: space spotted :)

**Review by JWittmeyer — APPROVED** [review]

File: upgrade_logic/business_objects/gateway.py (hunk 2 above)
Comment (SirDegraf) [reply upgrade_logic/business_objects/gateway.py]: Aye, my mistake

Comment (SirDegraf) [reply upgrade_logic/business_objects/gateway.py]: removed and updates submodule

**Pull request merged**